### PR TITLE
Bug 564420 conditions | move HC condition miner to new interfaces

### PR DESCRIPTION
**Pull request by eparovyshnaya**
- move licensing server coordinates detection close to proper settings
declaration
  - append tests for it

Signed-off-by: elena.parovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/internal/net/i18n/NetMessages.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.eclipse.passage.lic.internal.net.i18n;
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/internal/net/i18n/NetMessages.java:1]: again!

File: bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/internal/net/i18n/NetMessages.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+LicensingServerCoordinates_settings_are_blank=%s cannot be blank
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/internal/net/i18n/NetMessages.properties:1]: And again

Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/internal/net/i18n/NetMessages.properties:14]: Woha! Too early! Was not ready for review! )

Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/internal/net/i18n/NetMessages.properties:14]: Please, have a look now

**Pull request merged**